ugs-core/src/com/willwinder/universalgcodesender/utils/Settings.java

```diff
@@ -313,6 +315,16 @@ public void setSingleStepMode(boolean singleStepMode) {
         changed();
     }
 
+    public boolean isSingleBlockMode() {
+        return singleBlockMode;
+    }
+
+    public void setSingleBlockMode(boolean singleBlockMode) {
+        logger.log(Level.INFO, "Setting singleBlockMode to ".concat((singleBlockMode)?"true":"false"));
```

Comment:
It's a matter of taste of course, but I'd prefer to use `Boolean.toString(singleBlockMode)` here.